composer.json

```diff
@@ -5,6 +5,7 @@
   "license": "proprietary",
   "require": {
     "php": ">=7.2",
+    "laminas/laminas-filter": "^2.8",
```

Comment:
Please make it `spryker/laminas` if possible.

Reply:
Updated.